sunpy/net/base_client.py

```diff
@@ -172,15 +172,6 @@ class QueryResponseTable(QTable):
     display_keys = TableAttribute(default=slice(None))
     hide_keys = TableAttribute()
 
-    # This is a work around for https://github.com/astropy/astropy/pull/11217
-    # TODO Remove when min astropy version is > 4.2.1
```

Comment:
I think we need to change the version in setup.cfg to be 4.2.1 rather than 4.2.0.